src/Domain/Entity/Promotion.php

```diff
@@ -31,14 +34,21 @@ public function __construct(
         string $title,
         Synopses $synopses,
         string $url,
-        int $weighting
+        int $weighting,
+        bool $isSuperPromotion
     ) {
         $this->pid = $pid;
         $this->promotedEntity = $promotedEntity;
         $this->synopses = $synopses;
         $this->title = $title;
         $this->url = $url;
         $this->weighting = $weighting;
+        $this->isSuperPromotion = $isSuperPromotion;
+    }
+
+    public function getIsSuperPromotion(): bool
```

Comment:
isSuperPromotion()

Should we stick this at the bottom so the getters are in the same order as the constructor args? That's what all the other entities do.